x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -32,9 +32,43 @@ import {
   isTimeSeriesViewJob,
 } from '../../../../../../../common/util/job_utils';
 import { ML_APP_LOCATOR, ML_PAGES } from '../../../../../../../common/constants/locator';
+import { checkPermission } from '../../../../../capabilities/check_capabilities';
 
 const MAX_FORECASTS = 500;
 
+const DeleteForecastConfirm = ({ onCancel, onConfirm }) => (
+  <EuiConfirmModal
+    title={i18n.translate(
+      'xpack.ml.jobsList.jobDetails.forecastsTable.deleteForecastConfirm.title',
+      {
+        defaultMessage: 'Delete the selected forecast?',
+      }
+    )}
+    onCancel={onCancel}
+    onConfirm={onConfirm}
+    cancelButtonText={i18n.translate(
+      'xpack.ml.jobsList.jobDetails.forecastsTable.deleteForecastConfirm.cancelButton',
+      {
+        defaultMessage: 'Cancel',
+      }
+    )}
+    confirmButtonText={i18n.translate(
+      'xpack.ml.jobsList.jobDetails.forecastsTable.deleteForecastConfirm.confirmButton',
+      {
+        defaultMessage: 'Confirm',
+      }
+    )}
+    defaultFocusedButton="confirm"
+  >
+    <p>
+      <FormattedMessage
+        id="xpack.ml.jobsList.jobDetails.forecastsTable.deleteForecastConfirm.text"
+        defaultMessage="This will delete the forecast from the job permanently."
```

Comment:
Nit.

```suggestion
        defaultMessage="This will permanently delete the forecast from the job."
```

Reply:
Updated in d29de18790177bc0c0d512a43845f70518ab6dc7